src/java.base/share/classes/java/io/FileOutputStream.java

```diff
@@ -97,7 +97,10 @@ public class FileOutputStream extends OutputStream
 
     /**
      * Creates a file output stream to write to the file with the
-     * specified name. A new {@code FileDescriptor} object is
+     * specified name. {@linkplain java.nio.file##links Symbolic links}
+     * are automatically redirected to the <i>target</i> of the link.
+     * If the file or link target exists, then it will be truncated.
+     * A new {@code FileDescriptor} object is
```

Comment:
This wording gives the impression that a link can be truncated :-)

I think the second sentence needs to say that it truncates an existing file, otherwise creates a new file. Then follow with the sentence on sym link behavior.

Reply:
Fixed in a6c2168.